styles/styles.css

```diff
@@ -206,48 +207,89 @@ header {
 h1, h2, h3,
 h4, h5, h6 {
   font-family: var(--heading-font-family);
-  font-weight: var(--font-weight-bolder);
+  font-weight: var(--font-weight-bold);
   line-height: 1.25;
-  margin-top: 1em;
-  margin-bottom: .5em;
   scroll-margin: calc(var(--nav-height) + 1em);
 }
 
-h1 { font-size: var(--heading-font-size-xxl) }
-h2 { font-size: var(--heading-font-size-xl) }
-h3 { font-size: var(--heading-font-size-l) }
-h4 { font-size: var(--heading-font-size-m) }
-h5 { font-size: var(--heading-font-size-s) }
-h6 { font-size: var(--heading-font-size-xs) }
+h1 {
+  font-size: 34px;
```

Comment:
on bitdefender website the font-size for h1 is bigger, 56px or 50px, depending on the media query,  but on some pages is 40px, overridden by the page style. I would increase the default font size to something bigger, I think `--heading-font-size-xxl` would be a better choice. What do you think?

Reply:
there are media queries below, that change the size of h1 - this is the mobile font (the default value)

Reply:
ah, sorry, I got confused, they are not doing it mobile first